fv/vxlan_test.go

```diff
@@ -218,6 +218,14 @@ var _ = infrastructure.DatastoreDescribe("VXLAN topology before adding host IPs
 					policy.Spec.Selector = "has(host-endpoint)"
 					_, err = client.GlobalNetworkPolicies().Create(utils.Ctx, policy, utils.NoOptions)
 					Expect(err).NotTo(HaveOccurred())
+
```

Comment:
Why these changes to existing tests?

Reply:
See the full discussion about this in Slack at https://tigera.slack.com/archives/GUEHC5CKA/p1584559861114600 , which I kicked off by writing...

"Think I just found an issue of incompatibility w.r.t. existing behaviour; exposed by FV tests at https://github.com/projectcalico/felix/blob/master/fv/vxlan_test.go#L189 and https://github.com/projectcalico/felix/blob/master/fv/ipip_test.go#L176.  These tests have:

- a normal GNP policy defined, to allow egress to a particular IP, that will apply to host-endpoint=='true'
- WHEP defined on each node, with host-endpoint: true
- pre-DNAT ingress allow-all policy defined, applying to has(host-endpoint). 

Current behaviour:

- normal policy not implemented for WHEPs, so the normal GNP policy has no effect, and traffic between hosts is allowed.

Behaviour with WIP:

- the normal GNP policy now blocks all egress traffic from host to {other host or workload on other host}

1. Do you agree that this is the correct new behaviour, according to what we have so far agreed?
2. Should we perhaps specify something more explicit, to control whether existing normal and AOF policy starts applying to a WHEP?
"

Only @spikecurtis replied, and his view was that new behaviour is correct, and that we shouldn't add any knobs for precise back-compatibility with existing configuration.